lib/router/src/router.tsx

```diff
@@ -67,10 +68,18 @@ QueryLink.displayName = 'QueryLink';
 const QueryLocation = ({ children }: QueryLocationProps) => (
   <Location>
     {({ location }: RouteComponentProps): ReactNode => {
-      const { path } = queryFromString(location.search);
+      const { path, singleStory } = queryFromString(location.search);
       const { viewMode, storyId, refId } = parsePath(path);
 
-      return children({ path, location, navigate: queryNavigate, viewMode, storyId, refId });
+      return children({
+        path,
+        location,
+        navigate: queryNavigate,
+        viewMode,
+        storyId,
+        refId,
+        singleStory: singleStory === 'true',
```

Comment:
parseBoolean?

Reply:
It's in a different module, so I can't directly access it. Also `singleStory` is a new param which is not supposed to support `'1'`, but only `'true'`.